src/main/java/net/atos/client/zgw/ztc/ZTCClientService.java

```diff
@@ -235,7 +236,9 @@ public Besluittype readBesluittype(final UUID besluittypeUUID) {
      */
     @CacheResult(cacheName = ZTC_BESLUITTYPE)
     public List<Besluittype> readBesluittypen(final URI zaaktypeURI) {
-        return ztcClient.besluittypeList(new BesluittypeListParameters(zaaktypeURI)).getSinglePageResults();
+        return ztcClient.besluittypeList(new BesluittypeListParameters(zaaktypeURI)).getSinglePageResults().stream()
```

Comment:
DIt wordt gecachet! Je kunt hier dus zeker NIET naar LocalDate.now() gaan kijken. Kom op nou. :-(

Reply:
Dit was de verkeerde plek om dit op te lossen - opgelost in opvolgende commit.